scripts/systNtuple.py

```diff
@@ -5,11 +5,6 @@
 import fileinput
 import commands
 import time
-import glob
-import subprocess
-from os.path import basename
```

Comment:
don't we still need `os.path`?

Reply:
that line only imports basename, os is imported at the beginning